libs/eo/terms/src/lib/eo-terms.component.ts

```diff
@@ -99,16 +114,31 @@ import { WattSpinnerComponent } from '@energinet-datahub/watt/spinner';
   `,
 })
 export class EoTermsComponent {
+  private http = inject(HttpClient);
+  private termsService = inject(EoTermsService);
+  private authService = inject(EoAuthService);
+  private router = inject(Router);
+  private policyVersion$ = this.http.get<VersionResponse>(
+    '/assets/configuration/privacy-policy.json'
+  );
+
+  loadingPrivacyPolicyFailed = false;
+  privacyPolicy$ = this.policyVersion$.pipe(
+    switchMap((response) => {
+      this.termsService.setVersion(response.version);
+      return this.http.get('/assets/html/privacy-policy.html', { responseType: 'text' });
+    }),
+    catchError(() => {
+      this.termsService.setVersion(-1); // Is this needed anymore?
```

Comment:
@frontend-specialisten 
I would argue it is needed to ensure the version is kept in sync with the text of the privacy policy. Otherwise the following sequence is possible:

- client: fetch policy and version
- client: view the policy
- server: update the policy and version
- client: fetch new policy and fail to fetch version
- client: accept the new policy, but register the old version as accepted

It is unlikely to happen, but it seems like a nice safety net to have.